src/components/itemDetailsModal.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Button, Modal, ModalContent, Table } from 'semantic-ui-react'
+
+import { ONE_MB } from '../constants'
+import { setActiveItems } from '../actions/itemActions'
+
+import file from './css/details-modal.css'
```

Comment:
rename the css file to item-properties-modal.css.
rename this file(itemDetailModal.js) to "item-detail-modal.js" to keep the file naming same across the project